main.go

```diff
@@ -1195,7 +1199,7 @@ func generatePackageList(currentPackage string) PackageList {
 
 // Populates list of package wrappers, handles manual and automatic variation
 // If print, function will print information during processing
-func populatePackages(currentPackage string, onlyUpdates bool, onlyLatest bool, print bool) (PackageList, error) {
+func populatePackages(currentPackage string, onlyUpdates bool, onlyLatest bool, print bool, iconOverride bool) (PackageList, error) {
```

Comment:
I am pretty sure that the `iconOverride` argument is redundant with the `onlyUpdates` argument.

As I understand, the purpose of the `iconOverride` argument is to make sure that `populatePackages` returns all `PackageWrapper`s, not just ones that can be updated. However, that is also the purpose of the `onlyUpdates` argument. Why not just set `onlyUpdates` to `false` in `downloadIcons` and `overrideIcons` (which it already is)?

If you agree, we should explain what `onlyUpdates` is for in the function comment so we don't get into this mess again.

Reply:
I agree, I reverted these changes